alacritty/src/input/keyboard.rs

```diff
@@ -644,3 +642,10 @@ impl From<ModifiersState> for SequenceModifiers {
         modifiers
     }
 }
+
+/// Check that the text represents either C0 or C1 codepoint.
+fn is_c0c1_codepoint(text: &str) -> bool {
+    let codepoint = text.chars().next().map(u32::from).unwrap_or(0);
```

Comment:
This would mean that an empty string is a c0 or c1 codepoint. Which I'd say is inaccurate.

I also think that looking at chars/u32s is unnecessary here. Just getting `bytes()` should be fine since no valid unicode character (and `&str` is always valid unicode) can start with a C0/C1 character?

Reply:
Yeah, maybe checking for empty should be better on callee side.

Reply:
I still can't remove the len check due to compose input, since it could be an actual string in the end of the day.